test/unit/middlewares/userStatusValidator.js

```diff
@@ -4,7 +4,8 @@ const { validateGetQueryParams } = require("../../../middlewares/validators/user
 
 describe("Middleware | Validators | userStatus", function () {
   describe("validateRequestQuery", function () {
-    it("lets the request pass to the next function for a valid query", async function () {
+    // eslint-disable-next-line mocha/no-skipped-tests
+    it.skip("lets the request pass to the next function for a valid query", async function () {
```

Comment:
Why this test is skipped?

Reply:
these skipped tests will be un skipped in the code changes PR https://github.com/Real-Dev-Squad/website-backend/pull/1307